pkg/executables/kind.go

```diff
@@ -135,6 +136,22 @@ func (k *Kind) WithExtraDockerMounts() bootstrapper.BootstrapClusterClientOption
 	}
 }
 
+func (k *Kind) WithExtraCloudStackMounts(extraMounts []string) bootstrapper.BootstrapClusterClientOption {
+	return func() error {
+		if k.execConfig == nil {
+			return errors.New("kind exec config is not ready")
+		}
+
+		if len(extraMounts) == 0 {
+			return errors.New("no extraMounts found in the list")
```

Comment:
`extraMounts` does mean anything to someone reading the error. Rule of thumb, don't include variable names in errors.

With respect to this behavior, why does it matter that no mounts were specified - wouldn't that just be a noop?